problem_builder/migrations/0004_copy_course_ids.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+def copy_course_id_to_course_key(apps, schema_editor):
```

Comment:
If we try to merge this branch into master, it will cause some pain to have a different version here.  If we don't try to merge this branch into master, this version of the management command will disappear and will never have lived on the master branch, which may make it hard to dig up later on, if it's needed for historical reasons.  

My suspicion is that faking the migration will be the lowest amount of pain, but I'll defer to @feanil's judgment on which approach they'd prefer.